tools/ClassGen/NodeGen.cpp

```diff
@@ -58,6 +58,14 @@ int main(int argc, char **argv) {
   //                   Convolution / Pool / FC
   //===--------------------------------------------------------------------===//
 
+  BB.newNode("Pad")
+      .addInput("Input")
+      .addMember(MemberType::Unsigned, "Mode")
+      .addMember(MemberType::VectorSigned, "Pads")
+      .addMember(MemberType::Float, "Value")
+      .addResultFromCtorArg()
+      .setDocstring("Performs padding");
```

Comment:
Please add a more descriptive doc string. What is the semantics of the operator? What happens when negative values are used? What modes are supported? 

Also, can you please use ENUM instead of unsigned for the mode?

Reply:
I didn't use enum because `MemberType::Enum`  was not really supported by nodebuilder. I implemented a minimal support of `MemberType::Enum` in nodebuilder, that is still translated as `unsigned_t` currently in the generated API since as far as I know, there is no generic `enum` type in C++. The only thing that we know is:
> It is implementation-defined which integral type is used as the underlying type for an enumeration except that the underlying type shall not be larger than int unless the value of an enumerator cannot fit in an int or unsigned int.

Reply:
`Pad` doc string: done.